yarn-project/end-to-end/src/e2e_state_vars.test.ts

```diff
@@ -288,5 +289,66 @@ describe('e2e_state_vars', () => {
       expect(tx.data.forRollup!.rollupValidationRequests.maxBlockNumber.isSome).toEqual(true);
       expect(tx.data.forRollup!.rollupValidationRequests.maxBlockNumber.value).toEqual(new Fr(expectedMaxBlockNumber));
     });
+
+    it('checks max block number propagated by reading with our SharedMutablePrivateGetter in TestContract accessing AuthContract; before and after a decreasing then increasing delay change', async () => {
```

Comment:
Would prefer a shorter test name here. Not sure we need to be told exactly how, but more what it is testing, should be ok with 
```typescript
it('check propagation of max block number in private', async () => {
```